test/test_ulog2csv.py

```diff
@@ -46,7 +46,9 @@ def test_ulog2csv(self, test_case):
         messages = []
         output=tmpdir
         delimiter=','
-        ulog2csv.convert_ulog2csv(ulog_file_name, messages, output, delimiter)
+        time_s = 0
+        time_e = 1
```

Comment:
The unit test is failing, problably best to set this to 0.

```suggestion
        time_e = 0
```

```
                #get the index for row where timestamp exceeds or equals the required value
                time_s_i = np.where(d.data['timestamp'] >= time_s * 1e6)[0][0] if time_s else 0
                #get the index for row upto the timestamp of the required value
>               time_e_i = np.where(d.data['timestamp'] >= time_e * 1e6)[0][0] if time_e else len(d.data['timestamp'])
E               IndexError: index 0 is out of bounds for axis 0 with size 0
```